### PR TITLE
Error logging for send xcm to pallet-xcm

### PR DESCRIPTION
**Pull request by bkontur**

### CODE REVIEW

File: polkadot/xcm/pallet-xcm/src/lib.rs [hunk 1]
```diff
@@ -363,7 +363,10 @@ pub mod pallet {
 			let message: Xcm<()> = (*message).try_into().map_err(|()| Error::<T>::BadVersion)?;
 
 			let message_id = Self::send_xcm(interior, dest.clone(), message.clone())
-				.map_err(Error::<T>::from)?;
+				.map_err(|error| {
+					tracing::error!(target: "xcm::pallet_xcm::send", ?error, ?dest, ?dest, "XCM send failed with error");
```
Comment (bkontur) [inline polkadot/xcm/pallet-xcm/src/lib.rs:367]:
```suggestion
					tracing::error!(target: "xcm::pallet_xcm::send", ?error, ?dest, ?message, "XCM send failed with error");
```

**Review by acatangiu — APPROVED** [review]

File: polkadot/xcm/xcm-builder/src/process_xcm_message.rs [hunk 2]
```diff
@@ -58,7 +58,7 @@ impl<
 		let message = Xcm::<Call>::try_from(versioned_message).map_err(|_| {
 			log::trace!(
 				target: LOG_TARGET,
-				"Failed to convert `VersionedXcm` into `XcmV3`.",
+				"Failed to convert `VersionedXcm` into `xcm::prelude::Xcm`!",
```
Comment (franciscoaguirre) [inline polkadot/xcm/xcm-builder/src/process_xcm_message.rs:61]:
```suggestion
				"Failed to convert `VersionedXcm` into `xcm::latest::Xcm`!",
```

**Review by franciscoaguirre — APPROVED** [review]

File: polkadot/xcm/xcm-builder/src/process_xcm_message.rs (hunk 2 above)
Comment (bkontur) [reply polkadot/xcm/xcm-builder/src/process_xcm_message.rs:61]: prelude is here better than latest, because imports use `use xcm::prelude::*;`

**Pull request merged**